1906-10-05.xml

```diff
@@ -0,0 +1,7096 @@
+                            <cell><measure>12.60</measure></cell>
+                            <cell><measure>12.76</measure></cell>
+                            <cell><measure>7.20</measure></cell>
+                            <cell><measure>7.85</measure></cell>
+                            <cell><measure>97.20</measure></cell>
+                            <cell><measure>97.38</measure></cell>
+                            <cell><measure>91.41</measure></cell>
+                            <cell><measure>92.44</measure></cell>
+                            <cell><measure>13.10</measure></cell>
+                            <cell><measure>15.08</measure></cell>
+                            <cell><measure>18.20</measure></cell>
+                            <cell><measure>21.04</measure></cell>
+                        </row><row role="data">
+                            <cell><date when="1905-07-09">28</date></cell>
+                            <cell><measure>11.95</measure></cell>
+                            <cell><measure>12.33</measure></cell>
+                            <cell><measure>16.45</measure></cell>
+                            <cell><measure>17.68</measure></cell>
+                            <cell><measure>15.15</measure></cell>
+                            <cell><measure>15.62</measure></cell>
+                            <cell><measure>12.58</measure></cell>
+                            <cell><measure>12.58</measure></cell>
+                            <cell><measure>7.18</measure></cell>
+                            <cell><measure>7.83</measure></cell>
+                            <cell><measure>97.33</measure></cell>
+                            <cell><measure>97.38</measure></cell>
+                            <cell><measure>91.55</measure></cell>
+                            <cell><measure>92.43</measure></cell>
+                            <cell><measure>13.16</measure></cell>
+                            <cell><measure>15.07</measure></cell>
+                            <cell><measure>18.16</measure></cell>
+                            <cell><measure>21.04</measure></cell>
+                        </row><row role="data">
+                            <cell><date when="1905-07-09">29</date></cell>
+                            <cell><measure>11.95</measure></cell>
+                            <cell><measure>12.33</measure></cell>
+                            <cell><measure>16.27</measure></cell>
+                            <cell><measure>17.42</measure></cell>
+                            <cell><measure>15.10</measure></cell>
+                            <cell><measure>15.52</measure></cell>
+                            <cell><measure>12.55</measure></cell>
+                            <cell><measure>12.50</measure></cell>
+                            <cell><measure>7.05</measure></cell>
+                            <cell><measure>7.72</measure></cell>
+                            
```

Comment:
`<div type="section" feature="wire">`, then make the following item its own div within the section.